articles/advisor/advisor-reference-operational-excellence-recommendations.md

```diff
@@ -194,11 +194,11 @@ This cluster is not using ephemeral OS disks which can provide lower read/write
 
 Learn more about [Kubernetes service - UseEphemeralOSdisk (Use Ephemeral OS disk)](../aks/cluster-configuration.md#ephemeral-os).
 
-### Free and Standard pricing tiers for AKS control plane management
+### Free and Standard tiers for AKS control plane management
 
-This cluster has not enabled the Standard pricing tier with the Uptime SLA feature, and is limited to an SLO of 99.5%.
+This cluster has not enabled the Standard tier with the Standard tier feature, and is limited to an SLO of 99.5%.
```

Comment:
This cluster has not enabled the Standard tier which includes the Uptime SLA by default, and is ...